test/utils/trust/historical.beneficial.owner.mapper.spec.ts

```diff
@@ -29,7 +30,7 @@ describe('Historical Beneficial Owner page Mapper Service', () => {
         const mockFormData = {
           ...mockFormDataBasic,
           boId: '9999',
-          type: '0' as TrustHistoricalBeneficialOwnerType,
+          type: BeneficialOwnerType.Individual as TrustHistoricalBeneficialOwnerType,
```

Comment:
I feel like this could also be out of scope

Reply:
I believe this makes the code cleaner as at first glance you do not know what the string '0' means